app/views/components/tables/filtering/_filter_anonymous.html.erb

```diff
@@ -0,0 +1,22 @@
+<%- # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later -%>
+<!-- partial: components/filter_anonymous -->
+<form>
+	<fieldset>
+		<input type='radio' name='anonymous' id='radio-anonymous' value='true'>
+    <!--= on 'change' apply_radio_filter -->
+		<label for='radio-anonymous'>Anonymous</label>
+	</fieldset>
+		
+	<fieldset>
+		<input type='radio' name='anonymous' id='radio-not-anonymous' value='false'>
+    <!--= on 'change' apply_radio_filter -->
+    <label for='radio-not-anonymous'>Not-anonymous</label>
+	</fieldset>
+
+	<fieldset>
+		<input class='radio--both' type='radio' name='anonymous' id='radio-either' checked='checked' value=''>
+			<!--= on 'change' apply_radio_filter -->
+		<label for='radio-either'>Any</label>
+	</fieldset> 
+</form>
+<!-- partial end: components/filter_anonymous -->
```

Comment:
We need to decide on the exact UI but good work as always :)

Reply:
Do you think that having:

**Anonymous?**
* Yes
* No
* Either

would be better?

Reply:
I'm going to do it and send both versions for wendy to see what they think!